plugins/disable-top-blocks/GETSTARTED.md

```diff
@@ -0,0 +1,24 @@
+This package was bootstrapped with [@blockly/create-package](https://www.npmjs.com/package/@blockly/create-package).
```

Comment:
I see, I didn't realize this file was meant to be deleted. It could be useful if anyone makes other changes to this plugin in the future, but I guess we have general documentation for that and don't need to repeat it for each plugin.